include/vinecopulib/vinecop/class.hpp

```diff
@@ -81,11 +81,11 @@ class Vinecop
   void to_json(const std::string filename) const;
 
   // Methods modifying structure and/or families and parameters
-  void select_all(const Eigen::MatrixXd& data,
```

Comment:
Why?

Reply:
Because we need to create a copy inside the function anyway.

Reply:
Then you can remove `&`, but why remove the `const` ?

Reply:
Because instead of creating a copy explicitly, we just modify the copy created implictly^^